docs/install/index.md

```diff
@@ -141,11 +141,17 @@ $ pip install mxnet --pre
 
 </div> <!-- End of master-->
 <hr> <!-- pip footer -->
-Most MXNet versions offer an experimental MKL pip package that will be much faster when running on Intel hardware.
-Check the chart below for other options, refer to <a href="https://pypi.org/project/mxnet/">PyPI for other MXNet pip packages</a>, or <a href="validate_mxnet.html">validate your MXNet installation</a>.
+MXNet offers MKL pip packages that will be much faster when running on Intel hardware.
+Check the chart below for other options, refer to <a href="https://pypi.org/project/mxnet/">PyPI for other MXNet pip packages</a>, or <a href="validate_mxnet.html">validate your MXNet installation</a>. 
 
 <img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages-1.3.0.png" alt="pip packages"/>
 
+**NOTES:**
+
+*mxnet-cu92mkl* means the package is built with CUDA/cuDNN and MKL-DNN enabled and the CUDA version is 9.2.
+
+MKL pip package is experimental in some old versions of MXNet.
```

Comment:
Wouldn't it be more accurate to say "All MKL pip packages are experimental prior to version x.y.z."

Reply:
Thanks for the clarification. Will change the doc.